spec/src/main/asciidoc/entities.asciidoc

```diff
@@ -79,14 +79,25 @@ will result in a formatted amount (South African Rand) in the result:
 }
 ----
 
+List example:
+[source,java,numbered]
+----
+@Mutation
+public SuperHero trackHeroLongLat(@Name("name") String name,
+                                 @Name("coordinates") List<List<@NumberFormat("00.0000000 'longlat'") BigDecimal>> coordinates) throws UnknownHeroException {
+    // Here add the tracking
+    return superHero;
+}
+
 ==== Dates
 By default the date related scalars (DateTime, Date, and Time) will use a ISO format.
 
 - `yyyy-MM-dd\'T\'HH:mm:ss\'Z'` for DateTime
 - `yyyy-MM-dd` for Date
 - `HH:mm:ss` for Time 
 
-By adding the JSON-B annotation `@JsonbDateFormat` a user can change the format.
+By adding the `@DateFormat` annotation, or alternatively JSON-B annotation `@JsonbDateFormat`, a user can change the format. However, `@JsonbDateFormat` does not 
+support usage on `TYPE_USE`
```

Comment:
Same comment as above - we may need to specify a priority in case both annotations are used.